Source/Core/Core/ConfigManager.cpp

```diff
@@ -325,7 +325,7 @@ void SConfig::SaveNetworkSettings(IniFile& ini)
 
   network->Set("SSLDumpRead", m_SSLDumpRead);
   network->Set("SSLDumpWrite", m_SSLDumpWrite);
-  network->Set("SSLVerifyCert", m_SSLVerifyCert);
+  network->Set("SSLVerifyCertificates", m_SSLVerifyCert);
```

Comment:
We call SaveNetworkSettings every time any settings are going to be saved to Dolphin.ini, so I don't know how you've managed to avoid getting a Network section. (Or am I misinterpreting your comment?) I have a Network section with `SSLVerifyCert = false` even though I've never touched it.